runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -146,7 +146,8 @@ task googleCloudPlatformIntegrationTest(type: Test) {
           "--tempRoot=${dataflowTempRoot}",
   ])
 
-  include '**/*IT.class'
+  //include '**/*IT.class'
+  include '**/BigQueryToTableIT.class'
```

Comment:
Are there other tests that we are not including? Maybe this is only so we can run the test in the PR?